### DISCUSSION TITLE
posts/automate-lawn-garden/

### DISCUSSION THREAD

**Discussion by giscus** [techno-tim/techno-tim.github.io#262] [Comments]
# posts/automate-lawn-garden/

This week I finally decided to automate the watering of my lawn and garden without irrigation, here’s how…

https://technotim.live/posts/automate-lawn-garden/

**Comment by Robert-aug13** [comment]
Hi Tim!.. If you can please guide me. I have installed the prometheus-stack operator as per your guide..but having an issue in routing the alerts from Prometheus to Alert-Manager. Alerts are generating on Prometheus but not received on AlertManager. I used the same commands a month ago where I was able to set the below: alerting:
  alertmanagers:
  - static_configs:
    - targets:
      - 'localhost:9093'          # but now I dont see the alerting section in the configmap of prometheus-rules-0(statefulset pod).. I am wondering what has changed within a month. In the operator, prometheus.yml is replaced by configmap and also alertmanager.yml is set as a secret.(just for info)
